bindings/osrmcpy.py

```diff
@@ -59,7 +60,7 @@ def osrmc_error_errcheck(result, func, arguments):
 
 # Route
 lib.osrmc_route.restype = c.c_void_p
-lib.osrmc_route.argtypes = [c.c_void_p, c.c_void_p, c.c_void_p]
+lib.osrmc_route.argtypes = [c.c_void_p, c.py_object, c.c_void_p]
```

Comment:
I don't understand why we are passing a py_object to the C libosrmc library here. The C libosrmc library should not have a dependency on Python; it should be ABI and API stable and free-standing as much as possible.

For example when we use libosrmc from Haskell, how would we pass a py_object to the osrmc_route function?